packages/application-config/package.json

```diff
@@ -42,7 +42,7 @@
     "core-js": "^3.23.4",
     "cosmiconfig": "7.0.1",
     "dompurify": "^2.3.6",
-    "jsdom": "^19.0.0",
```

Comment:
Btw, this was also kind of "wrong" as we require the types for v16 `"@types/jsdom": "^16.2.14"`.